pandas/core/strings.py

```diff
@@ -197,8 +220,8 @@ def str_contains(arr, pat, case=True, flags=0, na=np.nan, regex=True):
 
 def str_startswith(arr, pat, na=np.nan):
     """
-    Return boolean array indicating whether each string starts with passed
-    pattern
+    Return boolean ``Series``/``array`` indicating whether each string in the ``Series``/``Index``
+    starts with passed pattern. Equivalent to :meth:`str.startswith(pat)`.
```

Comment:
the HTML doc will append `()` and this would show up as "equivalent to `str.startswith(pat)()`", probably best to remove the `(pat)` at the end